tests/_test_msui/test_wms_control.py

```diff
@@ -496,8 +496,12 @@ class TestWMSControlWidgetSetupSimple:
 
     @pytest.fixture(autouse=True)
     def setup(self, qtbot):
+        wc.WMS_SERVICE_CACHE = {}
         self.view = HSecViewMockup()
-        self.window = wc.HSecWMSControlWidget(view=self.view)
+        self.tempdir = tempfile.mkdtemp()
```

Comment:
Cleanup is missing. I'd prefer we'd just use pytest's builtin temp_path fixture instead, though.

Reply:
much better

the other similiiar usages can be refactored afterwads by a new issue.